### PR TITLE
docs(platform): add ariba labels to those component doc pages

### PR DESCRIPTION
**Pull request by droshev**
add ariba labels to those component doc pages

### CODE REVIEW

**Review by platon-rov — APPROVED** [review]

**Review by dimamarksman — APPROVED** [review]

**Review by olshansky — APPROVED** [review]

**Review by sKudum — APPROVED** [review]

**Review by N1XUS — APPROVED** [review]

File: apps/docs/src/app/platform/component-docs/platform-page-footer/platform-page-footer-header/platform-page-footer-header.component.html [hunk 1]
```diff
@@ -1,4 +1,8 @@
 <header>Page Footer Component</header>
+<div class="fd-info-labels">
+    <span title="SAP Ariba Component" class="fd-info-label fd-info-label--accent-color-8">SAP Ariba Design</span>
+    <span title="Fiori 3 Themeable Component" class="fd-info-label fd-info-label--accent-color-3">themeable</span>
```
Comment (InnaAtanasova) [inline apps/docs/src/app/platform/component-docs/platform-page-footer/platform-page-footer-header/platform-page-footer-header.component.html:4]: We have Fund-ngx components for this:
```
<fd-info-label label="Info Label" [color]="8" title="SAP Ariba Design"></fd-info-label>
<fd-info-label label="Info Label" [color]="3" title="themeable"></fd-info-label>
```

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
Please use fund-ngx components

File: apps/docs/src/app/platform/component-docs/platform-page-footer/platform-page-footer-header/platform-page-footer-header.component.html [hunk 2]
```diff
@@ -1,4 +1,8 @@
 <header>Page Footer Component</header>
+<div class="fd-info-labels">
```
[the remark this answers is not in the document]
Comment (droshev) [reply apps/docs/src/app/platform/component-docs/platform-page-footer/platform-page-footer-header/platform-page-footer-header.component.html:2]: no idea  😄  , it had taken it from another platform example. It has been removed now

File: apps/docs/src/app/platform/component-docs/platform-page-footer/platform-page-footer-header/platform-page-footer-header.component.html (hunk 1 above)
Comment (droshev) [reply apps/docs/src/app/platform/component-docs/platform-page-footer/platform-page-footer-header/platform-page-footer-header.component.html:4]: done

**Review by InnaAtanasova — APPROVED** [review]

**Comment by InnaAtanasova** [comment]
🚀

**Pull request merged**